src/UrlBuilder.php

```diff
@@ -2,34 +2,40 @@
 
 namespace Imgix;
 
-class UrlBuilder {
+class UrlBuilder
+{
+    private $currentVersion = '3.3.1';
 
-    private $currentVersion = "3.3.1";
     private $domain;
+
     private $useHttps;
+
     private $signKey;
-    private $includeLibraryParam=true;
 
-    const TARGET_WIDTHS = array(
+    private $includeLibraryParam = true;
+
+    public const TARGET_WIDTHS = [
         100, 116, 134, 156, 182, 210, 244, 282,
         328, 380, 442, 512, 594, 688, 798, 926,
         1074, 1246, 1446, 1678, 1946, 2258, 2618,
-        3038, 3524, 4088, 4742, 5500, 6380, 7400, 8192);
+        3038, 3524, 4088, 4742, 5500, 6380, 7400, 8192];
 
     // define class constants
     // should be private; but visibility modifiers are not supported php version <7.1
-    const TARGET_RATIOS = array(1, 2, 3, 4, 5);
+    public const TARGET_RATIOS = [1, 2, 3, 4, 5];
+
+    public const DPR_QUALITIES = [1 => 75, 2 => 50, 3 => 35, 4 => 23, 5 => 20];
 
-    const DPR_QUALITIES = array(1 => 75, 2 => 50, 3 => 35, 4 => 23, 5 => 20);
+    public const MIN_WIDTH = 100;
```

Comment:
AFAIK, adding `public` to these "private" class variables doesn't change much. So as long as that's the case and it's needed to make the linter happy, I don't have a problem with it.

Reply:
Yep! If not defined, `public` is assumed by PHP. 

> Properties declared without any explicit visibility keyword are defined as public.
> [PHP.net](https://www.php.net/manual/en/language.oop5.visibility.php#language.oop5.visibility-members)

Reply:
But better to be explicit, IMO.